cmd/http-server/startcmd/start.go

```diff
@@ -83,22 +83,30 @@ const (
 		" Alternatively, this can be set with the following environment variable (in CSV format): " +
 		agentHTTPResolverEnvKey
 
+	blocDomainFlagName      = "bloc-domain"
```

Comment:
note: we renamed the DID method to trustbloc.

Reply:
I will rename it next PR